backend/docs/API.md

```diff
@@ -257,6 +257,94 @@ You'll get
 }
 ```
 
+
+# Offer API
+
+## Get all offers created in the platform
+
+### GET
+
+One must provide the number of offers that should be sent.
+
+```
+/offers?size=<int>
+```
+
+#### Expected Reponse
+
+For size = 2
+
+```
+{
+    {
+        "offers": [
+            {
+            "id": 6,
+            "name": "Aula",
+            "description": "Present Perfect",
+            "extraInfo": "",
```

Comment:
the json fields are all in lower case: extrainfo, maxprice, etc.
Sorry, I forget to change the other examples in the doc.